src/login/logind-core.c

```diff
@@ -380,13 +381,42 @@ bool manager_shall_kill(Manager *m, const char *user) {
         return m->kill_user_processes;
 }
 
+int config_parse_n_autovts(const char *unit, const char *filename, unsigned line, const char *section, unsigned section_line, const char *lvalue, int ltype, const char *rvalue, void *data, void *userdata)
+{
+        unsigned *n = data;
+        unsigned o;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        r = safe_atou(rvalue, &o);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse number of autovts, ignoring: %s", rvalue);
+                return 0;
+        }
+
+        if (o > 15) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "A maximum of 15 autovts are supported, ignoring: %s", rvalue);
+                return 0;
+        }
+
+        *n = o;
+        return 0;
+}
```

Comment:
i figure you forgot to update src/login/logind-gperf.gperf accordingly to actually call this new funciton in the right case?

Reply:
Yep, looks like I got distracted part-way through or something :).  Thanks for reviewing!